src/index.ts

```diff
@@ -275,23 +279,47 @@ class LiskMigrator extends Command {
 
 					// Ask user to manually stop Lisk Core v3 and continue
 					const isLiskCoreV3Stopped = await cli.confirm(
-						"Please stop Lisk Core v3 to continue. Type 'yes' and press Enter when ready. [yes/no]",
+						"Please stop Lisk Core v3 to continue. Type 'yes' and press Enter when ready. [yes/no] ",
```

Comment:
Maybe since you know the old process can get the exact version here instead of `v3`? it can be `v3.1.0` or `v3.1.1` etc..